docker/postgres-server/scripts/entrypoint.sh

```diff
@@ -36,7 +36,6 @@ main () {
 
     # Customize pg_hba.conf
     local pg_hba="/home/certdir/pg_hba.conf"
-    sed -i 's/127.0.0.1\/32/0.0.0.0\/0/g' "${pg_hba}"
```

Comment:
I'm not sure what was the reason to have `127.0.0.1/32` in the source `pg_hba.conf` file and replace it with `0.0.0.0/0` later, so I replaced the value with the base `pg_hba.conf` file so we don't need to replace them. At the same time, I replaced `0.0.0.0/0` with `all` to support both `ipv4` and `ipv6` ips.